sound/soc/sof/sof-priv.h

```diff
@@ -179,6 +179,9 @@ struct snd_sof_dsp_ops {
 	/* DAI ops */
 	struct snd_soc_dai_driver *drv;
 	int num_drv;
+
+	/* platform-specific timeout */
+	int rom_init_timeout;
```

Comment:
that's not needed here.

Reply:
@plbossart  Sorry about that. fixed this one and the spurious line.